packages/abi-typegen/src/utils/collectBinFilePaths.test.ts

```diff
@@ -54,17 +54,20 @@ describe('collectBinFilePaths.ts', () => {
     expect(validateBinFile).toHaveBeenCalledTimes(1);
   });
 
-  test('should not collect bin files for Contracts', () => {
+  test('should collect bin files for Contracts', () => {
     const { validateBinFile } = mockDeps();
```

Comment:
We need to collect bin files for contracts now to export them via `*.hex.ts` files.

Reply:
Maybe we can make this opt-in via a CLI flag.

Reply:
Since the hex/bytecode will not be included automatically, we don't need a special flag.